stix2/test/constants.py

```diff
@@ -36,6 +45,7 @@
 MALWARE_KWARGS = dict(
     labels=['ransomware'],
     name="Cryptolocker",
+    description="A ransomware related to ..."
```

Comment:
Is `description` required? I'd rather keep these a minimal set, but it would be fine to have an additional set for a "realistic" Malware object.

Reply:
This has been fixed